### ISSUE TITLE
Spike specialized dropout layers

### ISSUE THREAD

**Issue by kimborgen** [kimborgen/bai#3]
The dropout layers cause the spikes (1) to scale to 1.42...
Que? When I remove the dropout its ok
spk_conv1 = self.dropout1(spk_conv1)

**Comment by kimborgen** [comment]
Ah, torch.dropout scales the spikes!
https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html
![image](https://github.com/kimborgen/bai/assets/13204226/af2e745d-f6e6-4b2e-a52b-f513f72af27a)

Because scaling is required for ANN but does not work for spikes, lets remove the dropout layers for now and research spike specific dropout methods.
